static/app/views/projectInstall/issueAlertOptions.tsx

```diff
@@ -76,290 +72,184 @@ type RequestDataFragment = {
-      frequency: 5,
-    };
-  }
-
-  setStateAndUpdateParents<K extends keyof State>(
-    state:
-      | ((
-          prevState: Readonly<State>,
-          props: Readonly<Props>
-        ) => Pick<State, K> | State | null)
-      | Pick<State, K>
-      | State
-      | null
-  ): void {
-    this.setState(state, () => {
-      this.props.onChange(this.getUpdatedData());
-    });
-  }
-
-  getEndpoints(): ReturnType<DeprecatedAsyncComponent['getEndpoints']> {
-    return [['conditions', `/projects/${this.props.organization.slug}/rule-conditions/`]];
-  }
-
-  onLoadAllEndpointsSuccess(): void {
-    const conditions = this.state.conditions?.filter?.((object: any) =>
-      Object.values(METRIC_CONDITION_MAP).includes(object?.id)
-    );
-
-    if (!conditions || conditions.length === 0) {
-      this.setStateAndUpdateParents({
-        conditions: undefined,
-      });
-      return;
-    }
-
-    const {intervalChoices, interval} = unpackConditions(conditions);
-    if (!intervalChoices || !interval) {
-      Sentry.withScope(scope => {
-        scope.setExtra('props', this.props);
-        scope.setExtra('state', this.state);
-        Sentry.captureException(
-          new Error('Interval choices or sent from API endpoint is inconsistent or empty')
-        );
-      });
-      this.setStateAndUpdateParents({
-        conditions: undefined,
-      });
-      return;
-    }
-
-    const newInterval =
-      this.props.interval &&
-      intervalChoices.some(
-        (intervalChoice: any) => intervalChoice[0] === this.props.interval
-      )
-        ? this.props.interval
-        : interval;
-
-    this.setStateAndUpdateParents({
-      conditions,
-      intervalChoices,
-      interval: newInterval,
-    });
-  }
-
-  renderBody(): React.ReactElement {
-    const issueAlertOptionsChoices = this.getIssueAlertsChoices(
-      this.state.conditions?.length > 0
-    );
-
-    return (
-      <Content>
-        <RadioGroupWithPadding
-          choices={issueAlertOptionsChoices}
-          label={t('Options for creating an alert')}
-          onChange={alertSetting => this.setStateAndUpdateParents({alertSetting})}
-          value={this.state.alertSetting}
-        />
-        {this.props.notificationProps &&
-          parseInt(this.state.alertSetting, 10) !== RuleAction.CREATE_ALERT_LATER && (
-            <IssueAlertNotificationOptions {...this.props.notificationProps} />
-          )}
-      </Content>
-    );
-  }
+        value={alertSetting.toString()}
+      />
+      {notificationProps && alertSetting !== RuleAction.CREATE_ALERT_LATER && (
+        <IssueAlertNotificationOptions {...notificationProps} />
+      )}
+    </Content>
+  );
 }
 
-export default withOrganization(IssueAlertOptions);
-
 const Content = styled('div')`
   padding-top: ${space(2)};
   padding-bottom: ${space(4)};
   display: flex;
   flex-direction: column;
-  gap: ${space(2)};
+  gap: ${space(4)};
 `;
 
-const CustomizeAlert = styled('div')`
```

Comment:
This was only really here so we could attach the `onClick` and call `stopPropagation()` in case a dropdown was clicked.

Instead i used a plain `<div>` to all `stopPropagation()` only if a dropdown is clicked. If the label is clicked then the RadioGroup will trigger it's normal onChange handler. Before there were two calls to `setStateAndUpdateParents({alertSetting});` but now we have only one place that's responsible for calling `setAlertSetting(selectedAlertSetting);`